### PR TITLE
Code cleaning, now i started to fix the activities

### PR DESCRIPTION
**Pull request by yairelboher**

### CODE REVIEW

File: app/src/main/java/com/example/sabonit/CategoriesActivity.java [hunk 1]
```diff
@@ -1,35 +1,33 @@
+/* ********* Imports: ********* */
```
Comment (liormoalem) [inline app/src/main/java/com/example/sabonit/CategoriesActivity.java:1]: decoration changes, seems good

Comment (eilonasher) [inline app/src/main/java/com/example/sabonit/CategoriesActivity.java:1]: perfect!

**Review by liormoalem — COMMENTED** [review]
great job, only decoration changes so from me its yes to apply

File: app/src/main/java/com/example/sabonit/CategoriesActivity.java [hunk 2]
```diff
@@ -40,9 +38,11 @@ public void onBackPressed() {
         showExitDialog();
     }
 
-    public void intentDepartment(View view) {
+    public void intentDepartment(View view)
+    {
         Intent intent = new Intent(this, ProductActivity.class);
-        switch (view.getId()) {
+        switch (view.getId())
```
Comment (eilonasher) [inline app/src/main/java/com/example/sabonit/CategoriesActivity.java:44]: great

File: app/src/main/java/com/example/sabonit/CategoriesActivity.java [hunk 3]
```diff
@@ -102,24 +103,28 @@ public void onComplete(@NonNull Task<QuerySnapshot> task) {
                 });
     }
 
-    public void logout(View view) {
+    public void logout(View view)
+    {
         Intent intent = new Intent(this, UserLoginActivity.class);
         intent.putExtra("Logout", 0);
         startActivity(intent);
     }
```
Comment (eilonasher) [inline app/src/main/java/com/example/sabonit/CategoriesActivity.java:112]: u didnt change some important things, u had to leave a general comment about WIP

**Review by eilonasher — COMMENTED** [review]
i accept

File: app/src/main/java/com/example/sabonit/CategoriesActivity.java (hunk 3 above)
Comment (yairelboher) [reply app/src/main/java/com/example/sabonit/CategoriesActivity.java:112]: well, you are right. next time

**Pull request merged**